samples/word/35-ranges/get-pages.yaml

```
@@ -0,0 +1,244 @@
+order: 4
+id: word-ranges-get-pages
+name: 'Work with pages, panes, and windows'
+description: 'Shows how to work with pages, panes, and windows APIs.'
+author: yilin4
+host: WORD
+api_set:
+    WordApiDesktop: '1.2'
+script:
+    content: |
+        $("#get-pages-selected-range").on("click", () => tryCatch(getPagesOfSelectedRange));
+        $("#get-pages-third-paragraph").on("click", () => tryCatch(getPagesOfThirdParagraph));
+        $("#get-pages-enclosing-viewport").on("click", () => tryCatch(getPagesEnclosingViewport));
+        $("#get-all-pages").on("click", () => tryCatch(getAllPages));
+        $("#setup").on("click", () => tryCatch(setup));
+
+        async function getPagesOfSelectedRange() {
+          await Word.run(async (context) => {
+            // Gets pages of the selection.
+            const pages: Word.PageCollection = context.document.getSelection().pages;
+            pages.load();
+            await context.sync();
+
+            console.log(pages);
+            // Log info for pages included in selection.
+            for (let i = 0; i < pages.items.length; i++) {
+              let page = pages.items[i];
+              page.load();
+              await context.sync();
+
+              console.log(`Index info for page ${i + 1} in the selection: ${page.index}`);
+              const range = page.getRange();
+              range.load();
+              await context.sync();
+
+              console.log("Text of that page in the selection:", range.text);
+            }
+          });
+        }
+
+        async function getPagesOfThirdParagraph() {
+          await Word.run(async (context) => {
+            // Gets the pages that the third paragraph is found on.
+            const paragraphs: Word.ParagraphCollection = context.document.body.paragraphs;
+            paragraphs.load();
+            await context.sync();
+
+            const paraThree = paragraphs.items[2];
+            const rangeOfParagraph = paraThree.getRange();
+            await context.sync();
+
+            const pages: Word.PageCollection = rangeOfParagraph.pages;
+            pages.load();
+            await context.sync();
+
+            console.log(pages);
+            // Log into for pages in range.
+            for (let i = 0; i < pages.items.length; i++) {
+              let page = pages.items[i];
+              page.load();
+              await context.sync();
+
+              console.log(`Index of page ${i + 1} that the third paragraph is found on: ${page.index}`);
+              const range = page.getRange();
+              range.load();
+              await context.sync();
+
+              console.log("Text of that page:", range.text);
+            }
+          });
+        }
+
+        async function getPagesEnclosingViewport() {
+          await Word.run(async (context) => {
+            // Gets the pages enclosing the viewport.
+
+            // Get the active window.
+            let activeWindow = context.document.activeWindow;
```

We switched from let to const partway through the snippet. Let's use const here and elsewhere.
```suggestion
            const activeWindow = context.document.activeWindow;
```